addons/web/static/src/legacy/js/core/utils.js

```diff
@@ -1138,12 +1149,21 @@ var utils = Object.assign({
         return [
             '&',
             ['res_model', '=', 'ir.ui.view'],
-            '|',
-            '|',
-            ['name', '=like', '%.assets\_%.css'],
-            ['name', '=like', '%.assets\_%.js'],
-            ['name', '=like', '%.report_assets\_%.css'],
-        ];
+        ].concat(Array(assetsPatterns.length * 2 - 1).fill('|'))
+        .concat(...assetsPatterns.map(pattern => [
+            ['name', '=like', pattern],
+            ['name', '=like', pattern + '.map'],
+        ]));
```

Comment:
This seems way overkill.

Remove the whole `assetsDomain()` method, it is only used once by our media dialog.
Then just add a domain in the media dialog (where it calls this method) to ignore the documents which have `/static/` in their path.
That heuristic should be enough, don't you think?

Reply:
Oh, you mean something like this https://github.com/odoo/odoo/pull/89151/files.
It seems to work indeed - even without the clause about `ir_ui_view`.

Reply:
Yep, thanks :+1:

Reply:
Yes Benoit, thank you.
My point was to just exclude everything coming from our asset engine, which basically always got prefixed by `/web/assets/`, see `_get_asset_template_url`.
Maybe this could be part of a comment mentioning that python method? It will be easier to figure where is your string exclusion (the domain) coming from **BUT** also it will allow to find this occurence of the code is someone is touching that `_get_asset_template_url` method one day in the python side, changing the URL or whatever. There is a good chance they blame the method name and find your occurence which is reflecting that method behavior.